waspc/cli/exe/Main.hs

```diff
@@ -127,8 +127,8 @@ printVersion = do
 -- TODO(matija): maybe extract to a separate module, e.g. DbCli.hs?
 dbCli :: [String] -> IO ()
 dbCli args = case args of
-  ["migrate-dev", migrationName] -> runDbCommand $ Command.Db.Migrate.migrateDev (Just migrationName)
   ["migrate-dev"] -> runDbCommand $ Command.Db.Migrate.migrateDev Nothing
+  "migrate-dev" : migrateArgs -> runDbCommand $ Command.Db.Migrate.migrateDev $ Just migrateArgs
```

Comment:
In the example above, they would be passed in like: `["--name", "FOO", "--create-only"]`

Reply:
Ha, this is getting a bit complex now!

Maybe we should just have it accept [String] then, and it can be empty if there are no arguments, instead of Maybe [String]? So [] would play role of Nothing.

Ideally, we would parse the strings and pass some nicely shaped params. That is a bit of work right now though, so I do agree with postponing it. We should probably switch the whole CLI to that applicative library and that would also help with this.

Reply:
I like your idea of moving the parsing into CLI. I can then pass in the `MigrateArgs` 👍🏻 Will do thnx